src/OpenLoco/src/Objects/ObjectIndex.cpp

```diff
@@ -347,44 +363,58 @@ namespace OpenLoco::ObjectManager
 
     static bool tryLoadIndex(const ObjectFolderState& currentState)
     {
+        Core::Timer loadTimer;
+
         const auto indexPath = Environment::getPathNoWarning(Environment::PathId::plugin1);
         if (!fs::exists(indexPath))
         {
+            Logging::verbose("Object index does not exist.");
             return false;
         }
-        std::ifstream stream;
-        stream.open(indexPath, std::ios::in | std::ios::binary);
-        if (!stream.is_open())
-        {
-            return false;
-        }
-        // 0x00112A14C -> 160
-        IndexHeader header{};
-        Utility::readData(stream, header);
-        if ((header.state != currentState) || (stream.gcount() != sizeof(header)))
+        FileStream stream;
+        stream.open(indexPath, StreamMode::read);
+        if (!stream.isOpen())
         {
+            Logging::error("Unable to load the object index.");
             return false;
         }
-        else
+
+        try
         {
-            if (reinterpret_cast<int32_t>(*_installedObjectList) != -1)
+            // 0x00112A14C -> 160
+            IndexHeader header{};
+            stream.readValue(header);
+            if (header.state != currentState)
             {
-                free(*_installedObjectList);
-            }
-            _installedObjectList = static_cast<std::byte*>(malloc(header.fileSize));
-            if (_installedObjectList == nullptr)
-            {
-                exitWithError(StringIds::unable_to_allocate_enough_memory, StringIds::game_init_failure);
+                Logging::verbose("Invalid object index header state.");
```

Comment:
Its not really invalid its just its changed due to a new/removed object. Okay i guess that means invalidated but feel it sounds a bit wrong.

Reply:
Object index out of date?

Reply:
Yeah that would be better

Reply:
Also maybe better to have this logged into info rather than verbose